src/test/java/com/portatlas/request/RequestParserTest.java

```diff
@@ -0,0 +1,81 @@
+package com.portatlas.request;
+
+import com.portatlas.request.Request;
+import com.portatlas.request.RequestParser;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.BufferedReader;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+import static org.junit.Assert.*;
```

Comment:
I think it's better to state which one you're importing exactly rather than doing `.*`